tests/functional/actions.js

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */ 
+
+const elements = require('./elements.js');
+const vpn = require('./helper.js');
+ 
+const connectionChangeNotification = async (status = 'true') => {
+    return await vpn.setSetting('connection-change-notification', status);
```

Comment:
the best is to have status as boolean and convert it to string here. Something like `status ? "true" : "false"`